DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/DeactivateDelegateController.cs

```diff
@@ -0,0 +1,96 @@
+﻿
+
+namespace DigitalLearningSolutions.Web.Controllers.TrackingSystem.Delegates
+{
+    using System.Collections.Generic;
+    using System.Globalization;
```

Comment:
please remove this, it is not required here

Reply:
Done